wolfcrypt/src/port/Renesas/renesas_tsip_util.c

```diff
@@ -103,74 +102,90 @@ static int tsip_CryptHwMutexUnLock(wolfSSL_Mutex* mutex)
 {
     return wc_UnLockMutex(mutex);
 }
-#ifdef WOLF_PRIVATE_KEY_ID
-/* Set client encrypted private key data.
+
+/* Set client encrypted public key data.
  * parameters:
  * ssl      WOLFSSL object
  * keyBuf  buffer holding wrapped key which Renesas key tool generated.
  * keyBufLen buffer length
  * keyType  0: RSA 2048bit, 1: RSA 4096bit, 2 ECC P256
  * return   0 on success, others on failure.
  */
-WOLFSSL_API int tsip_use_PrivateKey_buffer(struct WOLFSSL* ssl,
+#ifndef WOLFSSL_RENESAS_TSIP_CRYPTONLY
+WOLFSSL_API int tsip_use_PublicKey_buffer(WOLFSSL* ssl,
                                 const char* keyBuf, int keyBufLen, int keyType)
+#else
+WOLFSSL_API int tsip_use_PublicKey_buffer(TsipUserCtx *uc,
```

Comment:
I don't like the idea of us changing this API function definition if `WOLFSSL_RENESAS_TSIP_CRYPTONLY` is defined or not defined.  Seems like this will be confusing (or break builds) for users.  Can we have a separate function that gets compiled in when `WOLFSSL_RENESAS_TSIP_CRYPTONLY` is defined, with a different name than `tsip_use_PublicKey_buffer()`?

Reply:
Sure. Let me think about how to deal with this.

Reply:
Separate tsip_use_Public/PrivateKey_buffer to `TLS` and `Crypt`-Only